xml/deployment_yast_installer.xml

```diff
@@ -2699,7 +2699,7 @@ sle-live-patching 8c541494</screen>
     </para>
    </sect3>
    <sect3 xml:id="sec-s390-inst-reipl-reconnect-x">
-    <title>Using X to Connect</title>
+    <title>Using x to connect</title>
```

Comment:
Tricky one to catch, but 'X' should stay in capital letters.

Reply:
Wondering whether to handle that via a rule for single-character words or via the word list I already use. Thoughts?

Reply:
Your choice, but as I assume we would not have more single character terms than 'X' adding it to the word list is probably best.